contrib/debian/netdata.postinst.in

```diff
@@ -55,6 +55,15 @@ case "$1" in
     chown -R root:netdata /var/lib/netdata/www
     setcap cap_dac_read_search,cap_sys_ptrace+ep /usr/libexec/netdata/plugins.d/apps.plugin
 
+    chmod 4750 /usr/libexec/netdata/plugins.d/perf.plugin
+    chmod 4750 /usr/libexec/netdata/plugins.d/slabinfo.plugin
+    chmod 4750 /usr/libexec/netdata/plugins.d/cgroup-network
+    chmod 4750 /usr/libexec/netdata/plugins.d/nfacct.plugin
+    chmod 4750 /usr/libexec/netdata/plugins.d/ebpf_process.plugin
+
+    # Workaround for other plugins not installed directly by this package
+    chmod -f 4750 /usr/libexec/netdata/plugins.d/freeipmi.plugin
```

Comment:
@MrFreezeex I think you should put permission change for the freeipmi plugin in `netdata-plugin-freeipmi.postinst.in`.

Reply:
The chown in this postinst is removing the suid bit, so if you update the netdata package without the netdata-plugin-freeipmi package, the permission won't work.